### ISSUE TITLE
Enum Model generation is broken

### ISSUE THREAD

**Issue by codersam** [phiz71/vertx-swagger#74]
Hi,

I have problem generating model files which contains enums.
I used swagger-codegen 2.2.3 via gradle plugin.

It is currently also reproducible at http://editor.swagger.io/

**Steps to reproduce:**
1. Add enum model definition to  API, e.g.:
```yaml
  ErrorCode:
    type: string
    enum:
      - "REQUIRED_RESOURCE_ID_NOT_SET"
      - "REQUIRED_RESOURCE_NOT_FOUND"
      - "ALREADY_EXISTS"
      - "SOMETHING_ELSE"
```
2. Generate Server -> java-vertx.
3. Observer generated file for ErrorCode enum:
```java
package io.swagger.server.api.model;

import java.util.Objects;

public enum ErrorCode {
    {values&#x3D;[REQUIRED_RESOURCE_ID_NOT_SET, REQUIRED_RESOURCE_NOT_FOUND, ALREADY_EXISTS, SOMETHING_ELSE], enumVars&#x3D;[{name&#x3D;REQUIRED_RESOURCE_ID_NOT_SET, value&#x3D;&quot;REQUIRED_RESOURCE_ID_NOT_SET&quot;}, {name&#x3D;REQUIRED_RESOURCE_NOT_FOUND, value&#x3D;&quot;REQUIRED_RESOURCE_NOT_FOUND&quot;}, {name&#x3D;ALREADY_EXISTS, value&#x3D;&quot;ALREADY_EXISTS&quot;}, {name&#x3D;SOMETHING_ELSE, value&#x3D;&quot;SOMETHING_ELSE&quot;}]}, 
}
```

Kind regards,
Alex

**Comment by phiz71** [comment]
Hi, I solve the problem. A problem with a template...

However, you're using the java-vertx generator included into swagger-codegen 2.3.0. So I've also add a PR to fix it : https://github.com/swagger-api/swagger-codegen/pull/6428

**Comment by codersam** [comment]
Thank you very much for very quick fix.
Unfortunately I had no possibility to test it yet, as we decided to do current project without generation. 
But in next projects I will revisit this plugin for sure.
